controllers/useraccount/useraccount_controller.go

```diff
@@ -484,6 +491,28 @@ func (r *Reconciler) deleteUser(ctx context.Context, userAcc *toolchainv1alpha1.
 	return true, nil
 }
 
+// Returns `true` if the associated resources (configMap, role and role-binding) created for a user by console are deleted, `false` otherwise.
```

Comment:
It returns an error, not bool :)

Reply:
great catch, fixed in [a224e71](https://github.com/codeready-toolchain/member-operator/pull/568/commits/a224e71fffd02f8735473c20088feb02a2eaa6b0)